mmseg/ops/wrappers.py

```diff
@@ -24,8 +23,8 @@ def resize(input,
                         'the output would more aligned if '
                         f'input size {(input_h, input_w)} is `x+1` and '
                         f'out size {(output_h, output_w)} is `nx+1`')
-    if isinstance(size, torch.Size):
-        size = tuple(int(x) for x in size)
+    # if isinstance(size, torch.Size):
+    #     size = tuple(int(x) for x in size)
```

Comment:
If these are not needed, we may directly delete them.

Reply:
@xvjiarui If it is OK for training and testing, we could definitely remove these two lines.